addons/uh60_engine/functions/fnc_setup.sqf

```diff
@@ -36,4 +36,48 @@ player addEventHandler ["SelectedRotorLibActionPerformed", {
     [_target, (_enumNumber == 4)] call vtx_uh60_engine_fnc_wheelBrakes;
 }];
 
+//New aircraft module
```

Comment:
Should we handle the possibility of the aircraft already having been used in these setVariables? Or is that handles elsewhere